src/main/java/io/aiven/kafka/connect/http/config/HttpSinkConfig.java

```diff
@@ -272,6 +275,32 @@ private static void addBatchingConfigGroup(final ConfigDef configDef) {
 
     private static void addRetriesConfigGroup(final ConfigDef configDef) {
         int groupCounter = 0;
+        configDef.define(
+                KAFKA_RETRY_BACKOFF_MS_CONFIG,
+                ConfigDef.Type.LONG,
+                KAFKA_RETRY_BACKOFF_DEFAULT,
+                new ConfigDef.Validator() {
+                    @Override
+                    public void ensureValid(final String name, final Object value) {
+                        if (!(value instanceof Long)) {
+                            throw new ConfigException(name, value, "must be long");
+                        }
+                        final var longValue = (Long) value;
+                        if (longValue < KAFKA_RETRY_BACKOFF_DEFAULT) {
+                            throw new ConfigException(
+                                    KAFKA_RETRY_BACKOFF_MS_CONFIG, value,
+                                    "must be more or equals to " + KAFKA_RETRY_BACKOFF_DEFAULT);
+                        }
```

Comment:
Why can't it be less than the default? E.g. 3 seconds, 100 ms?

Also, there's `ConfigDef.Range` class for checking ranges :)